apps/common/mixins/serializers.py

```diff
@@ -228,7 +228,37 @@ def get_exclude_field_names(self):
         return fields_to_drop
 
 
+class DefaultValueFieldsMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.set_fields_default_value()
+
+    def set_fields_default_value(self):
+        if not hasattr(self, 'Meta'):
+            return
+        if not hasattr(self.Meta, 'model'):
+            return
+        model = self.Meta.model
+        for name, serializer_field in self.fields.items():
+            model_field = getattr(model, name, None)
+            if model_field is None:
+                continue
+            if not hasattr(model_field, 'field') \
+                    or not hasattr(model_field.field, 'default') \
+                    or model_field.field.default == NOT_PROVIDED:
+                continue
+            if name == 'id':
+                continue
+            default = model_field.field.default
+            if callable(default):
+                default = default()
+            serializer_field.default = default
```

Comment:
if not serializer_field.required and serializer_field.default == empty:
    serializer_field.default = default